pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -494,6 +484,26 @@ func (plugin *kubenetNetworkPlugin) Status() error {
 	return nil
 }
 
+// getContainerID returns the containerID that will be used to retrieve the pod IP.
+// For Docker, the container ID is the pod infra container's ID.
+// For rkt, the container ID is the API pod's UID.
+// TODO(yifan): Rename to getPodID.
+func (plugin *kubenetNetworkPlugin) getContainerID(pod *kubecontainer.Pod) kubecontainer.ContainerID {
```

Comment:
getNetworkContainerID? or getNetNSContainerID? 

I am bad at naming, but definitely not getContainerID.